gemp-lotr/gemp-lotr-async/src/main/web/src/updated/jCard/components/card.component.ts

```diff
@@ -0,0 +1,42 @@
+import { endDiv } from "../../../../src/types/html/html-janky.constants";
+import { createCardDisplayOptions } from "../types/card-display-options/card-display-options.factory";
+import { FoilPresentation } from "../types/foil-presentation.type";
+import { cardBase } from "./simple-card.component";
+
+const animatedFoil = 'foil.gif';
+const staticFoil = 'holo.jpg'
+
+export const renderCard = (imageUrl: string, text = '',  options = createCardDisplayOptions()): string => {
```

Comment:
This is the halfway point between "use inline functions to build HTML" and "use react".

Here we've taken all the things and just concatenated the reuslting HTML and then we return it. Whatever you do with it is up to you. In our case we're going to take it and use jQuery to append child, but this is more testable (even if I didn't write the tests yet) because we can check the long string of HTML for certain things and we can also do validity as well.